src/zed_cpu_ros.cpp

```diff
@@ -17,496 +17,483 @@
 #define HEIGHT_ID 4
 #define FPS_ID 5
 
-namespace arti
-{
+namespace arti {
```

Comment:
Can you run clang-format on this file with the config .clang-format in the repository so that I can see the diff?
Thanks.